source/blog/2017-04-05-say-hello-to-ovirt-4.1.1.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Say Hello to oVirt 4.1.1
+author: jmarks
+tags: 4.1.1, release notes, oVirt
+date: 2017-04-04 16:35:00 CET
+comments: true
+published: true
+---
+
+On March 22, the oVirt project released version 4.1.1, available for Red Hat Enterprise Linux 7.3, CentOS Linux 7.3, or similar.
+
+oVirt is the open source virtualization solution that provides an awesome KVM management interface for multi-node virtualization. This maintenance version is super stable and there are some nice new features.
+
+So what's new in oVirt 4.1.1?
+
+
+READMORE
```

Comment:
@mykaul @bproffitt @eedri 
Both files form part of the blog post. The top above is the snippet that you see on the main blog page, When you click READMORE you see the rest of the post, below. Brian, Eyal - What I did was simply duplicate a blog post and change the text. The outcome is not displayed in middleman. Is this the right way to do it?